distributed/scheduler.py

```diff
@@ -7476,6 +7482,9 @@ def decide_worker(
     of bytes sent between workers.  This is determined by calling the
     *objective* function.
     """
+    # TODO should it be a bounded fraction of `len(all_workers)`?
+    N_RANDOM_WORKERS: Py_ssize_t = 20
```

Comment:
Can we move this into the `else` block scope?  Mostly I'm optimizing to not distract readers until necessary, or to bound the distraction.

Reply:
Done